src/libraries/pkg/Microsoft.NETCore.Platforms/runtime.compatibility.json

```diff
@@ -1061,6 +1061,124 @@
     "any",
     "base"
   ],
+  "ios": [
+    "ios",
+    "unix",
```

Comment:
We should think about introducing some moniker that both osx and ios inherit from, in my other branch I used `darwin`.

There are quite a few cases in the libraries where the same implementation can be used on both osx and ios.

Reply:
@ericstj do you think it's worth it? The current practice seems to be that nothing sharable and  everything is included in a single package

Reply:
For the dependencies, we have a lot of RIDs that depend on other RIDs, the `<Parent>` element.

* linuxmint inherits from ubuntu
  * More complicatedly than other chains, because each version of linuxmint says which version of ubuntu it inherits from.
* ubuntu inherits from debian
* centos inherits from rhel
* ol inherits from rhel

It really just comes down to "when would someone publish a package with an asset targeting any-apple vs any?" If it's common, then a node in the RID graph is worthwhile. If it's never, then it's not needed.

I'm pretty sure that there's no real versioning here; so if there was demand for it later we can add it in with no real risk of breaking things.

Reply:
My take is that we can always expand the graph (add nodes) but never remove them so I'd start with the conservative approach and expand when we have specific needs for that